### PR TITLE
fix(stepper): update stepper styles

### PR DESCRIPTION
**Pull request by arjunlalb**
## Description
Update stepper component styles to 
1. Position the action buttons absolutely.
2. Make content itself scrollable

### CODE REVIEW

File: projects/components/src/stepper/stepper.component.scss [hunk 1]
```diff
@@ -25,6 +25,19 @@
         color: white;
       }
     }
+
+    .mat-stepper-horizontal {
```
Comment (anandtiwary) [inline projects/components/src/stepper/stepper.component.scss:29]: would this break styling for any existing stepper usage?

Comment (anandtiwary) [reply projects/components/src/stepper/stepper.component.scss:29]: cc @23nobody

**Review by anandtiwary — APPROVED** [review]
LGTM

File: projects/components/src/stepper/stepper.component.scss (hunk 1 above)
Comment (arjunlalb) [reply projects/components/src/stepper/stepper.component.scss:29]: @anandtiwary I tested on existing stepper usage and it worked fine. Will wait for @adisreyaj and @23nobody to review.

**Review by 23nobody — APPROVED** [review]

**Review by adisreyaj — APPROVED** [review]

**Pull request merged**